Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -231,7 +231,7 @@ wxMenuBar* CFrame::CreateMenu()
 
 	// Tools menu
 	wxMenu* toolsMenu = new wxMenu;
-	toolsMenu->Append(IDM_MEMCARD, _("&Memcard Manager (GC)"));
+	toolsMenu->Append(IDM_MEMCARD, _("&Memory Card Manager (GC)"));
```

Comment:
I agree that Memory Card is better than Memcard, but there are several other places in Dolphin where Memcard is used. I would rather wait until after 5.0 and change them all over in one go.